### ISSUE TITLE
Two typos

### ISSUE THREAD

**Issue by tpristavec** [koalaverse/hands-on-machine-learning-with-r#16]
**Date on the book's first page**: 2019-06-25

In **11 Random Forests, 11.4 Hyperparameters, 11.4.2. mtry**, last sentence of first paragraph:

> When there are many relevant predictors, a lower mtry might _**perfrom**_ better.

In **12 Gradient Boosting, 12.2 How boosting works, 12.2.1 A sequential ensemble approach**, second sentence of fourth paragraph:

> The idea behind boosting is that each model in the sequence slightly improves upon the **_perfroamnce_** of the previous one (essentially, by focusing on the rows of the training data where the previous tree had the largest errors or residuals).

**Comment by tpristavec** [comment]
And another:

In **12 Gradient Boosting, 12.2 How boosting works, 12.2.2 Gradient descent**, first sentence of first paragraph:

> Many algorithms in regression, including decision trees, focus on minimizing **_the some_** function of the residuals; most typically the SSE loss function, or equivalently, the MSE or RMSE (this is accomplished through simple calculus and is the approach taken with least squares).

**Comment by bradleyboehmke** [comment]
These are good catches @tpristavec. We have corrected these in the PDF version. Note that the online version will not be updated until we finalize the PDF.
